http/src/request/guild/create_guild/mod.rs

```diff
@@ -123,7 +129,7 @@ pub enum GuildChannelFields {
 }
 
 impl GuildChannelFields {
-    pub fn id(self) -> ChannelId {
+    pub fn id(&self) -> ChannelId {
```

Comment:
This would be a breaking change unfortunately, are we targeting this for the next major version of 0.3?

Reply:
Targeted 0.3